data.rs

```diff
@@ -0,0 +1,170 @@
+use crate::PSP22Error;
```

Comment:
We might think about adding a mod-level documentation here, explaining why _this_ representation and how it's supposed to be integrated into other contracts.

Reply:
Will be added soon (tm) ;)